careers.html

```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
 	<section class="career-view">
 		<div class="container">
 
-			<div id="datawrangler">
-				<h2>Data Wrangler</h2>
+			<div id="frontend">
+				<h2>Frontend Developer (Intermediate)</h2>
 				<div class="row">
 					<div class="col-md-6">
-						<p class="career-desc">We’re looking for an intermediate data wrangler / developer to help innovate storytelling in the field of journalism. The data wrangler will be involved in pushing the boundaries of data-driven storytelling, working with journalists to explore datasets to identify story ideas, analysing large datasets on corporates to support investigative journalists and be involved in general data analysis projects related to storytelling.</p>
-						<p class="career-desc">The candidate requires general scripting skills in python although extensive experience developing enterprise systems is not expected. The position will require some travel and the ability to work on intensive projects on tight deadlines. Projects will be relatively short and the candidate will be need to be creative about solving problems. He or she should also be confident in presenting their work to audiences.</p>
-						<p class="career-desc">The position is based in Cape Town but some assignments will be based elsewhere for short periods of time.</p>
+						<p class="career-desc">We’re looking for an intermediate, experience frontend developer to help us build useful, intuitive, open-source websites and tools.</p>
+						<p class="career-desc"><a href="/tools/">Our projects</a> range from small, single-page embeddable tools to large websites with rich content. The succesful applicant will help design and implement these projects and ensure they're helpful and user-friendly.</p>
+
+						<p class="career-desc">Candidates should be well-versed in modern frontend development tools and best-practices, be strong advocates for a good user experience, and able to work well independently and collaboratively.</p>
+						<p class="career-desc">We're particularly interested in candidates that share our passion for working to help South Africans make more informed decisions.</p>
```

Comment:
I realise more and more that I am completely out of touch with what drives developers. One technical challenge that may appeal to a more senior intermediate is the ability to conceptualise how to maximise the value that users get from a site.